src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -94,55 +94,49 @@
 
   private static final String INSTANCE_ID_FIELD_NAME = "instanceId";
+  private static final String INSTANCE_IDS_ENRICH_PARAM_NAME = "instanceIds";
 
   private static final String SKIP_SUPPRESSED_FROM_DISCOVERY_RECORDS = "skipSuppressedFromDiscoveryRecords";
-
   private static final String SUPPRESS_FROM_DISCOVERY = "suppressFromDiscovery";
-
-  private static final String INSTANCE_IDS_ENRICH_PARAM_NAME = "instanceIds";
-
   private static final String DELETED_RECORD_SUPPORT_PARAM_NAME = "deletedRecordSupport";
 
   private static final String START_DATE_PARAM_NAME = "startDate";
-
   private static final String END_DATE_PARAM_NAME = "endDate";
 
   private static final String TEMPORARY_LOCATION = "temporaryLocation";
-
   private static final String PERMANENT_LOCATION = "permanentLocation";
-
   private static final String EFFECTIVE_LOCATION = "effectiveLocation";
 
   private static final String CODE = "code";
 
   private static final String HOLDINGS = "holdings";
 
   private static final String INVENTORY_ITEMS_AND_HOLDINGS_ENDPOINT = "/inventory-hierarchy/items-and-holdings";
-
   private static final String INVENTORY_UPDATED_INSTANCES_ENDPOINT = "/inventory-hierarchy/updated-instance-ids";
 
   private static final String ERROR_FROM_STORAGE = "Got error response from %s, uri: '%s' message: %s";
-  public static final int RESPONSE_CHECK_ATTEMPTS = 30;
-
-  protected final Logger logger = LogManager.getLogger(getClass());
-
-  public static final MarcWithHoldingsRequestHelper INSTANCE = new MarcWithHoldingsRequestHelper();
-  private final Vertx vertx;
+  private static final String DOWNLOAD_INSTANCES_MISSED_PERMISSION = "Cannot download instances due to lack of permission, permission required - inventory-storage.inventory-hierarchy.updated-instances-ids.collection.get";
+  private static final String ENRICH_INSTANCES_MISSED_PERMISSION = "Cannot get holdings and items due to lack of permission, permission required - inventory-storage.inventory-hierarchy.items-and-holdings.collection.post";
+  private static final String INVENTORY_STORAGE = "inventory-storage";
 
+  private static final int DATABASE_FETCHING_CHUNK_SIZE = 5000;
   private static final int REREQUEST_SRS_DELAY = 2000;
-
   private static final int POLLING_TIME_INTERVAL = 500;
   private static final int MAX_WAIT_UNTIL_TIMEOUT = 1000 * 60 * 20;
   private static final int MAX_POLLING_ATTEMPTS = MAX_WAIT_UNTIL_TIMEOUT / POLLING_TIME_INTERVAL;
 
-  private  MetricsCollectingService metricsCollectingService = MetricsCollectingService.getInstance();
-  private InstancesService instancesService;
+  public static final MarcWithHoldingsRequestHelper INSTANCE = new MarcWithHoldingsRequestHelper();
+
+  private final Vertx vertx;
   private final WorkerExecutor saveInstancesExecutor;
   private final Context downloadContext;
 
+  private MetricsCollectingService metricsCollectingService = MetricsCollectingService.getInstance();
+  private InstancesService instancesService;
```

Comment:
Just corrected the order of variables declaration to have better readability and view.